### PR TITLE
Update version to 24015-00 using new PS script.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
* Checking-in a simpler version of a corefx script to update our repo to use the latest package version.
* Updating from rc3-23931-00 to rc3-24015-00

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
Missed some new tests, rebasing and updating.

**Comment by iamjasonp** [comment]
Buddy built and builds in TFS for me, LGTM

File: UpdateDependencies.ps1 [hunk 1]
```diff
@@ -0,0 +1,52 @@
+﻿#
```
Comment (roncain) [inline UpdateDependencies.ps1]: Nit: I would have preferred the ps1 and the version changes to be separate commits within this PR.  It makes cherry picking easier if you don't have to take both.  No need to change for this PR.

File: src/Common/test-runtime/LatestDependencies/project.json [hunk 2]
```diff
@@ -1,74 +0,0 @@
-{
```
Comment (roncain) [inline src/Common/test-runtime/LatestDependencies/project.json]: Question only: did the automatic tool delete this.  Was it manually deleted?  Don't block on the question.

**Comment by roncain** [comment]
LGTM

File: src/Common/test-runtime/LatestDependencies/project.json (hunk 2 above)
Comment (StephenBonikowsky) [inline src/Common/test-runtime/LatestDependencies/project.json]: I deleted it, Hong said it is no longer used or needed.
